roles/json-filter/defaults/main.yaml

```diff
@@ -1,12 +1,16 @@
 ---
 # Git branches
-json_filter_git_branch: develop
+json_filter_git_branch: dev-rabbit
 
 # Paths to services folder
 json_filter_workdir: /opt/json-filter
 
 # Variables
-json_filter_rmq_host: rabbit
-json_filter_rmq_port: 5672
-json_filter_rmq_login: mquser
-json_filter_rmq_pass: mqpass
+json_filter_rabbit_host: rabbit
+json_filter_rabbit_port: 5671
+json_filter_rabbit_user: mquser
+json_filter_rabbit_pw: "{{ lookup('aws_ssm', 'mqpass', region='eu-central-1') }}"
```

Comment:
Variables must be in path `/ENV_CLASS/ENV_NAME/mqpass`

Reply:
what we are should use for ENV_CLASS?